lib/turn.rb

```diff
@@ -0,0 +1,25 @@
+require './lib/turn'
+require './lib/card'
+require './lib/deck'
+require './lib/round'
+
+class Turn
+    attr_reader :cards, :turns
+
+  def initialize(cards, turns)
+    @cards = card
+    @turns = turn
+  end
+
+  def guess
+    @turns.guess #need this to return guess
+  end
```

Comment:
No need for this method. Based on my feedback above, in order to have a method `guess` that will return the guess that the Turn object was initialized with, you'll need to create an attr_Reader to read that value.